lib/ansible/modules/network/junos/junos_netconf.py

```diff
@@ -139,7 +139,7 @@ def map_params_to_obj(module):
     for key, value in iteritems(obj):
         # validate the param value (if validator func exists)
         validator = globals().get('validate_%s' % key)
-        if all((value, validator)):
+        if callable(validator):
```

Comment:
if `netconf_port` has no value in playbook eg: `netconf_port:` the `and value` condition will bypass the validation and will result in sending `None` value of port to the device, which is not right.

I think current logic to identify if validation function is implemented or not is more extendible instead of having `if` clause for each `key`